src/ios/CDVMediaStream.m

```diff
@@ -182,7 +182,7 @@ - (void)getUserMedia:(CDVInvokedUrlCommand*)command
             NSString *uuid = [[NSUUID UUID] UUIDString];
             [audioTracks setObject:uuid forKey:@"id"];
             [audioTracks setObject:@"audio" forKey:@"kind"];
-            [audioTracks setObject:device.deviceType forKey:@"description"];
+            [audioTracks setObject:device.localizedName forKey:@"label"];
```

Comment:
@jcesarmobile can you change `description` to `label` in the video section just above this code?

Reply:
Didn't want to change it as it's already changed in issue3 branch, so I just fixed the iOS 9 crash problem on this PR.
But will close this and send a new PR agains issue3 branch